docs/modules/secret-operator/pages/secretclass.adoc

```diff
@@ -36,6 +36,28 @@ CAUTION: Attributes of the certificate (such as the expiration date, fingerprint
 
 xref:scope.adoc[Scopes] are used to populate the claims (such as `subjectAlternateName`) of the provisioned certificates.
 
+==== Certification lifetime
```

Comment:
```suggestion
==== Certificate lifetime
```

Reply:
Most of this should eventually apply to all kinds of secrets, not just certificates.

Reply:
But maybe we don't need to generalize the docs quite yet